Fix f90wrap install

fekad opened the pull request:
# Notes for user:

- Basically the only requirement is python 3 with ASE (numpy is already required by ASE). I think each user should decide which version of numpy and ase wants to isnstall. Of course the minimum versions should be mentioned but not nessesarry to maintain the requirement.txt to force the users.


## Installing the `f90wrap` package:

- if pip is installed
  ```
  pip install src/f90wrap
  ```

- if pip is not available:
  ```
  cd src/f90wrap
  python setup.py install
  ```

# Developer notes:

- These notes can be added to the readme file.


## Cloning the repo:
  ```
  git clone --recursive https://github.com/libAtoms/QUIP.git

  ```

  Alternativle you can clone it wothout the submodules:
  ```
  git clone https://github.com/libAtoms/QUIP.git
  ```

  And you can initialise the submodules when its nedded:
  ```
  cd QUIP
  git submodule update --init --recursive
  ```

## Fixing/updating the version f90wrap:

  ```bahs
  cd src/f90wrap
  ```
  ```bash 
  git checkout <commit> 
  ```
  OR 
  ```bash 
  git checkout master

  ```
  Updating the version in the `QUIP` repository:
  ```
  cd ../..
  git add src/f90wrap
  git commit -m "updating the version of f90wrap"
  ```

Commit:
restoring travis config

Commit:
udating pip install command

Commit:
syntax highlights, cleanup

Commit:
requirements.txt became unnecessary

Commit:
testing fixed commit for f90wrap

Commit:
updating the version of f90wrap

fekad commented:
I did similar changes for building the docker image for quip:
https://github.com/libAtoms/quip-docker/commit/df272df17f98aabbcbe3dcab06a3b9a86eb17a35

jameskermode commented:
Thank you, this looks great! Can you add the notes above to the `README.md` directly, then we can merge all in one go?

stenczelt committed:
update readme

add update of submodules to f90wrap installation and remove reference to GAP-filler, add Adam to list of contributors

stenczelt reviewed (approved):
I agree with the changes and have updated minor things in the readme in addition to them.

stenczelt committed:
adding developer notes to readme
